docs/tools/swift.pod

```diff
@@ -1,30 +1,111 @@
 =pod
 
+=encoding utf8
+
 =head1 NAME
 
-swift - the amazingly new programming language
+swift -- General-purpose programming language
 
 =head1 SYNOPSIS
 
-B<swift> [-help] [I<input-filename> [I<program-arguments>]]
+To invoke the Swift REPL (Read-Eval-Print-Loop):
+
+=over
+
+B<swift>
+
+=back
+
+To execute a Swift program:
+
+=over
+
+B<swift> program.swift -- <arguments>
+
+=back
+
+To work with the Swift Package Manager:
+
+=over
+
+B<swift> build|package|run|test [options]
+
+=back
+
+To invoke the Swift compiler:
+
+=over
+
+B<swiftc> [options] <inputs>
+
+=back
+
+A list of supported options is available through the "-help" option:
+
+=over
 
-B<swiftc> [B<-emit-object>|B<-emit-assembly>|B<-emit-library>]
-  [-help]
-  B<-o> I<output-file>
-  I<input-filenames>
+B<swift> -help
 
-The full list of supported options is available via "swift -help" or "swiftc -help".
+B<swift> build -help
+
+B<swiftc> -help
+
+=back
 
 =head1 DESCRIPTION
 
-B<Swift> is a new, high performance systems programming language.  It has a clean
-and modern syntax, and offers seamless access to existing C and Objective-C code
-and frameworks, and is memory safe (by default).
+Swift is a general-purpose programming language built using a modern approach to
+safety, performance, and software design patterns.
+
+The goal of the Swift project is to create the best available language for uses
+ranging from systems programming, to mobile and desktop apps, scaling up to
+cloud services. Most importantly, Swift is designed to make writing and
+maintaining I<correct> programs easier for the developer. To achieve this goal,
+we believe that the most obvious way to write Swift code must also be:
+
+=head2 SAFE
```

Comment:
Are these best as headings, or as @CodaFi experimented with, might they work better as bolded text? After comparing the two, I'm inclined to think the latter. In part, I think it's because these don't fit in with the other headings (Synopsis, Description, Bugs, See Also, Home Page...Expressive (huh?)).